assets/models.py

```diff
@@ -0,0 +1,132 @@
+from django.db import models
+from django.db.models import Case, When, Q, BooleanField, Value
+from multiselectfield import MultiSelectField
+
+
+class AssetManager(models.Manager):
+    def get_queryset(self):
+        """A QuerySet to add an annotation to specify if an Asset is complete or not"""
+        return super().get_queryset().annotate(is_complete=Case(When(Q(
+            Q(name__isnull=False),
+            Q(department__isnull=False),
+            Q(purpose__isnull=False),
+            Q(owner__isnull=False),
+            Q(private__isnull=False),
+            Q(personal_data__isnull=False),
+            Q(data_subject__isnull=False),
+            Q(data_category__isnull=False),
+            Q(Q(data_category__contains="others", data_category_others__isnull=False) |
+              ~Q(data_category__contains="others")),
+            Q(recipients_category__isnull=False),
+            Q(recipients_outside_eea__isnull=False),
```

Comment:
Recipients category and Recipients outside EEA isn't marked as required on the spreadsheet I'm looking at.